PDF-style-guide.md

```diff
@@ -0,0 +1,74 @@
+# PDF spec guidelines
+
+The purpose of the PDF specs is to be readable by a wide range of people. They should not assume
+- knowledge of common (functional) programming syntax and idioms or
+- familiarity with another Cardano-specific resource, other than other specifications.
+
+## Haskell/Agda-isms
+
+We want to avoid syntax that is not commonly used outside of the
+languages we use. While this is really inconvenient sometimes, the
+more we use such notations the more we need to introduce them,
+increasing the barrier to entry.
+
+## Hiding pieces of code
+
+Hiding pieces of the underlying Agda code can be a useful tool to
+accomplish these objectives, but it is important that this is not
+overused. Any code whose meaning depends on hidden code must be
+explained by prose. However, it is always preferable if a figure can
+be fully understood without extra explanation.
+
+## Avoiding complexity
+
+- reduce the number of dependencies as much as possible, which can
+  often be done by generalizing
+- move (simple) definitions from being defined right next to their
+  usage to where they would naturally be defined
+
+## Alignment
+
+For better visual presentation, we should align figures properly. This
+might sometimes not be possible while also maintaining other
+constraints. Two cases where we usually want to align things are:
+- multiple lines of the form `a = b`, where those lines are short enough
+- a long expression of the form `x = a1 + ... + an`, broken into
+  multiple lines and align the `=` and `+` signs (also applies to any
```

Comment:
```suggestion
  multiple lines and the `=` and `+` symbols are aligned (also applies to any
```

(minor grammar fix)

Reply:
Actually, I've reformatted this list slightly in a different way